geode-core/src/main/java/org/apache/geode/internal/cache/backup/UnixScriptGenerator.java

```diff
@@ -50,6 +50,8 @@ public void writeCopyDirectoryContents(final BufferedWriter writer, final File b
   @Override
   public void writeCopyFile(final BufferedWriter writer, final File backup, final File original)
       throws IOException {
+    writer.write("mkdir -p '" + original.getParent() + "'");
```

Comment:
This looks like a product fix. Shouldn't that have its own jira ticket and test for it?